### DISCUSSION TITLE
How to create a shortcut into desktop launcher?

### DISCUSSION THREAD

**Discussion by ilcpm** [LSPosed/LSPosed#1466] [Q&A] [answered]
Maybe this is a silly question, but truely I can't find a way to add shortcut on desktop

when I installed LSP, it create a shortcut on OnePlus Launcher, but I changed to Microsoft Launcher later. And now, I can't find any way to add a shortcut to Microsoft Launcher but have to change back to OnePlus Launcher to open LSP.

Is there any button to do that which I didn't notice?

**Comment by yujincheng08** [comment, marked as answer]
You need to delete the shortcut from oneplus launcher first
